Data/Primitive/PrimArray.hs

```diff
@@ -480,6 +502,8 @@ unsafeThawPrimArray (PrimArray arr#)
   = primitive (\s# -> (# s#, MutablePrimArray (unsafeCoerce# arr#) #))
 
 -- | Read a primitive value from the primitive array.
+--
+-- /Note:/ this function does not\ do bounds checking.
```

Comment:
What's this backslash doing here?

Reply:
Apologies, that's an accidental keyboard hit, let me fix it.